docs/DeveloperGuide.md

```diff
@@ -709,10 +709,34 @@ Make sure to use the `exit` command or the close button to save data while closi
 
 1. _{ more test cases …​ }_
 
-#### Sort Feature
-
-1. Delete the data file (`./data/internshipdata.json`) before launching the app to populate the app with sample data.
-
+### Finding internships
+For all the following test cases:
+
+Prerequisites: Delete the data file (`./data/internshipdata.json`) before launching the app to populate the app with sample data.
+Then, list all internships using the `list` command.
+1. Filtering by company name
+   1. Test case: `find withall /com Amazon`<br>
+       Expected: 1 internship with the name 'Amazon' is shown. The status message shows how many internships were listed.
+
+   1. Test case: `find withall /com TikTok`<br>
+       Expected: No internships are shown. The status message shows how many internships were found.
```

Comment:
consistent message. stick with "The status message shows how many internships were listed." or "The status message shows how many internships were found."